content/pt-br/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md

```diff
@@ -0,0 +1,586 @@
+  Para o caso de _startup_ ou _liveness probe_, se ao menos a verificação de `failureThreshold` falhou, 
+  o Kubernetes tratará o contêiner como não íntegro e dispara a reinicialização do contêiner específico.
+  O kubelet leva em consideração `terminationGracePeriodSeconds` atribuído ao contêiner.
+  Para uma falha de _readiness probe_, o kubelet continua rodando o contêiner que falhou, 
+  e também continua rodando mais verificações; devido à verficação ter falhado, 
+  o kubelet atribui `false` a [condição](/docs/concepts/workloads/pods/pod-lifecycle/#pod-conditions)
+  no Pod.
+* `terminationGracePeriodSeconds`: configura um período de graça para que o kubelet espere antes de disparar o desligamento do contêiner falho, 
+  então força o gerenciador de contêiner a parar este contêiner. 
+  O padrão é herdar o valor `Pod-level` para `terminationGracePeriodSeconds` 
+  (30 segundos se não especificado), e o valor mínimo é 1.
+  Veja [_probe-level_ `terminationGracePeriodSeconds`](#probe-level-terminationgraceperiodseconds) para mais detalhes.
+ 
+ {{< note >}}
+Nas versões anteriores ao Kubernetes 1.20, o campo `timeoutSeconds` 
+não era respeitado por verificações de execução:
+verificações continuavam executando indefinidamente, até além do prazo final configurado,
+até que um resultado fosse devolvido.
+
+Essa falha foi corrigida na versão Kubernetes v1.20. Você pode estar confiando 
+no comportamento anterior, mesmo sem perceber, pois o tempo limite padrão é de 1 segundo. 
+Como administrador de cluster, você pode desabilitar o [portal de funcionalidades]
+(/docs/reference/command-line-tools-reference/feature-gates/) `ExecProbeTimeout` 
+(setando ela para `false`) em cada kubelet para restaurar o comportamento de versões 
+mais antigas, então remova essa substituição uma vez que todas as verficações 
+de execução no cluster estiverem com um valor atribuído para `timeoutSeconds`.
+Se você tem Pods que são impactados pelo padrão de tempo de 1 segundo,
+você deve atualizar o tempo limite de verificação para que esteja pronto para a
+eventual remoção desse portal de funcionalidades.
+
+Com a correção dessa falha, para verificações de execução, no Kubernetes `1.20+` 
+com o contêiner runtime `dockershim`, o processo dentro do contêiner pode continuar 
+rodando mesmo após a falha de verificação por causa do `timeout`.
+{{< /note >}}
+
+{{< caution >}}
+Implementações incorretas de _readiness probes_ podem resultar em um número sempre 
+crescente de processos no contêiner e escassez de recursos se isso for deixado desmarcado.
+{{< /caution >}}
+
+### verificações HTTP
+
+[Verificações HTTP](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#httpgetaction-v1-core)
+têm campos adicionais que podem ser setados em `httpGet`:
+
+* `host`: Nome do host para se conectar, por padrão é o IP do Pod. Você provávelmente vai quer
```

Comment:
typo: "provávelmente"

"Você provavelmente vai querer" instead of "Você provávelmente vai quer"